src/meetings.ts

```diff
@@ -114,6 +115,12 @@ export default function (zoomRequest: ReturnType<typeof request>) {
       params: params,
     })
   }
+  const ListPastMeetings = function (meetingId) {
+    return zoomRequest<ListPastMeetingsResponse>({
+        method: 'GET',
+        path: `/past_meetings/${meetingId}/instances`,
```

Comment:
nit: I kinda liked how things were route-organized before (all the `/user` things were together, then all the `/meetings` are listed), so we could preserve that and move `/past_meetings` to the end, but no strong feelings.

Reply:
sg!